scripts/genomes_cluster.py

```diff
@@ -0,0 +1,72 @@
+
```

Comment:
I like to name my scripts with action verbs when possible. Hence names like `select_strains.py`, `extract_sequences.py`, `concat_metadata.py`. I'd suggest renaming this script to something like `extract_cluster_fastas.py`.